src/components/DocumentationTopic/DocumentationNav/VersionToggle.vue

```diff
@@ -0,0 +1,283 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <NavMenuItemBase class="nav-menu-setting version-container">
+    <div :class="{ 'version-toggle-container': manyVersions }">
+      <!-- Faux element to get width of select, with current element-->
+      <select
+        class="version-dropdown version-sizer"
+        ref="version-sizer"
+        aria-hidden="true"
+        tabindex="-1"
+      >
+        <option selected>{{versionModel}}</option>
+      </select>
+      <!-- Faux element is above -->
+      <label
+        :for="versionList ? 'version-toggle' : null"
+        class="nav-menu-setting-label"
+      >Version:</label>
+      <select
+        v-if="versionList.length >1 "
+        id="version-toggle"
+        class="version-dropdown nav-menu-link"
+        :style="`width: ${adjustedWidth}px`"
+        v-model="versionModel"
+        @change="pushRoute(versionRoute)"
+      >
+         <option v-for="version in versionList"
+        v-bind:key="version"
+        :value="version">
+        {{version}}
+        </option>
```

Comment:
```suggestion
      <option
        v-for="version in versionList"
        :key="version"
        :value="version"
      >
        {{ version }}
      </option>
```

There are some spacing issues here, and we can just use the short hand of `v-bind` for `key` like the one you used for `value`